Squest/settings.py

```diff
@@ -30,6 +30,12 @@
 DB_HOST = os.environ.get('DB_HOST', '127.0.0.1')
 DB_PORT = os.environ.get('DB_PORT', '3306')
 LDAP_ENABLED = str_to_bool(os.environ.get('LDAP_ENABLED', False))
+OPENID_ENABLED = str_to_bool(os.environ.get('OPENID_ENABLED', False))
+OPENID_BTN_TEXT = os.environ.get('OPENID_BTN_TEXT', 'OpenID Login')
+SOCIAL_AUTH_OIDC_OIDC_ENDPOINT = os.environ.get('OIDC_ENDPOINT', '')
```

Comment:
We should keep python variable name and env var align to be consistent.

Also all of this should be placed in the doc.

I would go with the same prefix for all variable that concern OIDC.

Reply:
I can do that. Should have that shortly.